hacs.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "map-card",
     "render_readme": true,
-    "filename":"map-card.js"
```

Comment:
As mentioned if the file is part of a release, hacs should fetch it from the release (I hope it can, I'm not a HACS user).

Reply:
Ah, guessing this means I'll need to update the workflow https://github.com/nathan-gs/ha-map-card/blob/main/.github/workflows/release.yml to npm install and npm build as part of it?

Poss something like
https://github.com/MindFreeze/ha-sankey-chart/blob/master/.github/workflows/release.yml

Reply:
Yes, but I'll do so (want to introduce devenv here as well).

Reply:
Ah, cool. Otherwise it look like the workflow from the boilerplate card the sankey plugin uses should be pretty easy to swap in (Which gets the artefact added to the release).

Not used devenv before so interested to see what devenv Gh workflow stuff looks like! Cheers for merging 👍 (hopefully i've not broken your builds 😬 )